Opserver.Core/Data/Dashboard/Providers/DashboardDataProvider.cs

```diff
@@ -83,6 +83,8 @@ public Node GetNodeByHostname(string hostName)
         public abstract Task<List<GraphPoint>> GetUtilizationAsync(Volume volume, DateTime? start, DateTime? end, int? pointCount = null);
         public abstract Task<List<DoubleGraphPoint>> GetPerformanceUtilizationAsync(Volume volume, DateTime? start, DateTime? end, int? pointCount = null);
 
+        public virtual async Task<bool> UpdateServiceAsync(Node node, string serviceName, Data.Dashboard.NodeService.Action action) => false;
+
```

Comment:
Instead of putting this on the base as a bit of a bait and switch (in the current layout we get a bunch of inactive buttons on every non-WMI provider), let's instead create an interface: `IServiceControlProvider` which has only the one method, I'd do:
```c#
public interface IServiceControlProvider
{
    Task<bool> ControlServiceAsync(Node node, string serviceName, Data.Dashboard.NodeService.Action action);
}
public static class ServiceControlProviderExtensions
{
    public static bool HasServiceControl(this Node node) => node?.DataProvider is IServiceControlProvider;
}
```
...and implement the interface itself on WMI, the only provider to support it this far.